core/orchestrator.go

```diff
@@ -945,6 +949,16 @@ func (n *LivepeerNode) upscale(ctx context.Context, req worker.UpscaleMultipartR
 	return n.AIWorker.Upscale(ctx, req)
 }
 
+func (n *LivepeerNode) AudioToText(ctx context.Context, req worker.AudioToTextMultipartRequestBody) (*worker.TextResponse, error) {
+
+	resp, err := n.AIWorker.AudioToText(ctx, req)
```

Comment:
@eliteprox based on the code in https://github.com/livepeer/ai-worker/blob/0908b518eb1203617a3a68625d3326679a636439/worker/worker.go#L253-L305 we could simply forward the return values without performing the extra error check right 🤔?

Reply:
Removed unnecessary logic